pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -121,10 +122,10 @@ def on_epoch_end(self, epoch, logs=None):
         current = logs.get(self.monitor)
         stop_training = False
         if current is None:
-            logging.info(
-                'Early stopping conditioned on metric `%s` '
-                'which is not available. Available metrics are: %s' %
-                (self.monitor, ','.join(list(logs.keys()))), RuntimeWarning)
+            warnings.warn(
```

Comment:
`warnings.warn` and `logging.warning` are a completely different stream with different consequences so please be sure that you want this behaviour... You may consider also raise it as Error

Reply:
hmmm I'm not familiar with this, but I noticed that the previous code used warnings.warn a lot, I thought this was kind of best-practice.

Reply:
I believe that it was related to using `print`, in such case, it is better to `warnings`
I do not say that is wrong, in this case it is good, just to be aware of it... :]